example/src/advanced/index.html.js

```diff
@@ -4,20 +4,23 @@ import stylesheet from '@kaliber/build/lib/stylesheet'
 import javascript from '@kaliber/build/lib/javascript'
 import polyfill from '@kaliber/build/lib/polyfill'
 import App from './App?universal'
-import { pickFromRouteMap } from '@kaliber/routing'
+import { pick } from '@kaliber/routing'
 import { routeMap } from './routeMap'
 
 // TODO, should we add another example directory for advanced and only show the base path concept here?
 // - Yes
 const basePath = '/advanced' // TODO: Get from file name, maybe also add a public path example
 
 Index.routes = {
-  match(location) {
-    const ok = { status: 200 }
-    const result = pickFromRouteMap(location.pathname.replace(`${basePath}`, ''),
-      [routeMap, ok],
-      [routeMap.notFound, { status: 404 }],
-      [routeMap.articles.article.notFound, { status: 404 }]
+  match(location, req) {
+    const language = req.acceptsLanguages('nl', 'en') || 'nl'
```

Comment:
Cool, dit ken ik niet.

Reply:
Ik ook niet, haha, maar nu kennen we het allebei :-D